plat/common/bootinfo_fdt.c

```diff
@@ -85,38 +85,23 @@ static void fdt_bootinfo_mem_mrd(struct ukplat_bootinfo *bi, void *fdtp)
 		ukplat_bootinfo_crash("Could not add free memory descriptor");
 }
 
-static void fdt_bootinfo_cmdl_mrd(struct ukplat_bootinfo *bi)
+static void fdt_bootinfo_cmdl_init(struct ukplat_bootinfo *bi, void *fdtp)
```

Comment:
In the commit msg: `s/rmd/mrd/`.
Also maybe `const void *fdtp`?

Reply:
> In the commit msg: s/rmd/mrd/.

Ok

>  Also maybe const void *fdtp?

I'm following the original author's convention 😉  Saying that, `fdtp` is redundant in most of these functions, as it can be obtained from the `bi` parameter already, so I can add a commit to clean that up.

Reply:
Makes sense.